### PR TITLE
React app with semantic ui

### PR DESCRIPTION
**Pull request by shibeknight**
Issue: React app with Semantic UI #4

- Installed Semantic UI Framework using yarn add
- Importing semantic CSS file in the app's entry point index.js
- Added a Top menu bar in App.js using Semantic's examples 
- Created a basic layout using Semantic's Grid collection, 2 grids were created, the first one is just a single column that resizes based on device width and the second contains multiple stackable columns that adjust responsively as well
- All the components were added to App.js to showcase the basic layout using semantic, will be refactored into separate components accordingly later on

### CODE REVIEW

**Review by LHBO19 — DISMISSED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,196 @@
...
+              alt="placeholder"
+            />
+          </Menu.Item>
+          <Menu.Item
+            name="home"
+            // active={activeItem === "home"}
+            // onClick={this.handleItemClick}
+            // link
+            // as={Link}
+            // to="/"
```
Comment (avelinoschz) [inline src/App.js]: Why is this commented?

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,196 @@
...
+          >
+            Home
+          </Menu.Item>
+
+          <Menu.Item
+            name="profile"
+            // active={activeItem === "testimonials"}
+            // onClick={this.handleItemClick}
+            // as={Link}
+            // to="/profile"
```
Comment (avelinoschz) [inline src/App.js]: Ditto

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,196 @@
...
+          >
+            Profile
+          </Menu.Item>
+
+          <Menu.Item
+            name="sign-in"
+            // active={activeItem === "sign-in"}
+            // onClick={this.handleItemClick}
+            // as={Link}
+            // to="login"
```
Comment (avelinoschz) [inline src/App.js]: Ditto

File: src/App.js (hunk 1 above)
Comment (shibeknight) [reply src/App.js]: Added and commented so it is easier to implement the behavior for the menu items later as it needs routing to be installed first and a function to display the active item correctly when the view changes.

**Review by avelinoschz — COMMENTED** [review]
Why in every import you changed to double quotes?
Also saw, some king of inconsustency in some spacing and some quoting/double quoting

Choose and stick to some code style guide. For example: https://github.com/airbnb/javascript/tree/master/react

File: src/App.js (hunk 1 above)
Comment (avelinoschz) [reply src/App.js]: Try not to push commented code to the repo, unless absolutely necessary, to keep the code cleaner.

**Comment by shibeknight** [comment]
> Why in every import you changed to double quotes?
> Also saw, some king of inconsustency in some spacing and some quoting/double quoting
> 
> Choose and stick to some code style guide. For example: https://github.com/airbnb/javascript/tree/master/react

Code style will be fixed using a linter(Airbnb) to be added as a different issue.

File: src/App.js (hunk 1 above)
Comment (shibeknight) [reply src/App.js]: Fixed

File: src/App.js (hunk 2 above)
Comment (shibeknight) [reply src/App.js]: Fixed

File: src/App.js (hunk 3 above)
Comment (shibeknight) [reply src/App.js]: Fixed

**Review by avelinoschz — APPROVED** [review]

**Review by LHBO19 — APPROVED** [review]

**Comment by avelinoschz** [comment]
[React app with Semantic UI](https://trello.com/c/uWRikXA4/4-react-app-with-semantic-ui)

**Pull request merged**